packages/mobile/src/verify/komenci.ts

```diff
@@ -143,8 +143,7 @@ export function* fetchOrDeployMtwSaga() {
     // user already has a verified MTW
     const verifiedMtwAddress = yield call(fetchVerifiedMtw, contractKit, walletAddress)
     if (verifiedMtwAddress) {
-      // TODO: Address this
-      // yield put(doVerificationFlow(true))
+      yield put(succeed())
```

Comment:
Not succeeding here was causing already verified users to get stuck in the verification flow with no actions to do.